packages/graphql/schemas/schema.graphql

```diff
@@ -1276,19 +1276,6 @@ type FileParts implements Node {
   relative: String!
 }
 
-enum FrontendFrameworkEnum {
-  angular
-  nextjs
-  nuxtjs
-  react
-  reactscripts
-  svelte
-  vue2
-  vue3
-  vueclivue2
-  vueclivue3
-}
-
```

Comment:
These are no longer specific strings, since third parties can be called other things. We lose (some) type safety here... unavoidable.